tensorflow/core/platform/cloud/retrying_file_system.cc

```diff
@@ -39,14 +41,18 @@ bool IsRetriable(Status status) {
 
 Status CallWithRetries(const std::function<Status()>& f) {
   int attempts = 0;
+  int delay;
   while (true) {
+    delay = 1 << attempts;
     attempts++;
     auto status = f();
     if (!IsRetriable(status) || attempts >= kMaxAttempts) {
       return status;
     }
     LOG(ERROR) << "The operation resulted in an error and will be retried: "
                << status.ToString();
+    delay = delay > maximumBackoffSeconds ? maximumBackoffSeconds : delay;
```

Comment:
I wonder if this line and the line 46 could be merged also using std::min, e.g
int delay = std::min(1 << attempts, kMaximumBackoffSeconds) ? This could result in shorter code.